src/varlinkctl/varlinkctl.c

```diff
@@ -668,6 +698,100 @@ static int verb_call(int argc, char *argv[], void *userdata) {
                 } else
                         r = 0;
 
+                if (arg_exec && r == 0) {
+                        (void) sd_notify(/* unset_environment= */ false, "READY=1");
+
+                        _cleanup_free_ char *formatted = NULL;
+                        r = sd_json_variant_format(reply, arg_json_format_flags, &formatted);
```

Comment:
`arg_json_format_flags` is OFF by default??
I think it is better to fall back to `SD_JSON_FORMAT_NEWLINE` when OFF, and maybe better to mention that in the man page?

Reply:
We're passing this as stdin for the chainloaded executable to read, i.e. parsed by machine. Seems rather useless to break lines?

Reply:
NEWLINE means '--json=short', and newline is only at the end of the serialized string.
PRETTY means '--json=pretty', and multiple newline and indentations are inserted.
OFF means `--json=off`, and `sd_json_variant_format()` will fail in that case.